### PR TITLE
remove devman.html #3292

### LINKED ISSUE #3292: Remove devman.html
Can remove it as #2764 is done.

### PR DESCRIPTION
**Pull request by XianHuiKoh**
Fixes #3292

### CODE REVIEW

**Comment by LowWeiLin** [comment]
Sorry for not mentioning in the issue, some references to devman.html need to be updated.

image

**Comment by XianHuiKoh** [comment]
so should I change the second reference to glossary.md and for the first reference, overview.md or design.md?

**Comment by LowWeiLin** [comment]
Yes

File: src/main/webapp/dev/spec.html [hunk 1]
```diff
@@ -1208,7 +1208,7 @@ <h2 id="supplementaryrequirements-security">Security</h2>
 
     <h1 id="glossary">Glossary</h1>
     <p>
-        Refer to the Glossary in the <a href="devman.html#glossary"
+        Refer to the Glossary in the <a href="glossary.md"
```
Comment (LowWeiLin) [inline src/main/webapp/dev/spec.html]: This link for glossary.md is broken. Try using an absolute path?

**Comment by XianHuiKoh** [comment]
ignore latest commit

**Comment by LowWeiLin** [comment]
Merging without the last commit

**Pull request closed without merging**